client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java

```diff
@@ -222,8 +221,8 @@ private void doWatch(int numAttemptsSoFar) {
         }
 
         final Revision lastKnownRevision = latest != null ? latest.revision() : Revision.INIT;
-        final CompletableFuture<Latest<T>> f = doWatch(client, projectName, repositoryName, lastKnownRevision,
-                                                       WATCH_TIMEOUT_MILLIS);
+        final CompletableFuture<Latest<T>> f = doWatch(client, projectName, repositoryName, lastKnownRevision
+        );
```

Comment:
This can't be one line? if so, How about splitting before `lastKnownRevision`?

Reply:
Oops :sweat_smile: